library/lib/platforms/switch/switch_audio.cpp

```diff
@@ -23,7 +23,8 @@ namespace brls
 
 #define QLAUNCH_PID 0x0100000000001000
 #define QLAUNCH_MOUNT_POINT "qlaunch"
-#define QLAUNCH_BFSAR_PATH "qlaunch:/sound/qlaunch.bfsar"
+#define ROMFS_MOUNT_POINT "romfs"
+#define BFSAR_PATH ":/sound/qlaunch.bfsar"
```

Comment:
I don't like having the `:` in the "path" because technically it's not a valid character for POSIX paths, which we are kinda trying to stay compliant with. I would rather have the `:` added in the sprintf like `%s:%s`.

Reply:
Fixed. :+1: